pyFAST/fastfarm/FASTFarmCaseCreation.py

```diff
@@ -2022,15 +2194,397 @@ def FF_slurm_submit(self, qos='normal', A=None, t=None, delay=4):
                     # Submit the script to SLURM
                     fname = f'runFASTFarm_cond{cond}_case{case}_seed{seed}.sh'
 
-                    options = f"--qos='{qos}' "
+                    options = ''
                     if A is not None:
                         options += f'-A {A} '
                     if t is not None:
-                        options += f'-t {t} '
+                        options += f'-t {t}'
 
-                    sub_command = f"sbatch {options}{fname}"
+                    sub_command = f"sbatch {options} {fname}"
                     print(f'Calling: {sub_command}')
                     _ = subprocess.call(sub_command, cwd=self.path, shell=True)
                     time.sleep(delay) # Sometimes the same job gets submitted twice. This gets around it.
 
 
+    def Create_Mannbox(self, Lm, gamma):
```

Comment:
Thanks for creating this function. It is a really good capability to add to this case generation tool.